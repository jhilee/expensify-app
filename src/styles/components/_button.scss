.button {
    background: $button-blue;
    border: none;
    color: white;
    display: inline-block;
    font-size: $font-size-large;
    font-weight: 300;
    line-height: 1;
    text-align: center;
    padding: $s-size;
    text-decoration: none;
}

.button--link {
    background: none;
}

.button--remove {
    @extend .button;
    background: #b3b0b0
}