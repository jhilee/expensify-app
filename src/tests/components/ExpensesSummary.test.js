import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render correct statments totalling 1 expenses', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={4500} />);
    expect(wrapper).toMatchSnapshot();  
}); 

test('should render correct statments totalling 2 expense', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={2} expensesTotal={56000034} />);
    expect(wrapper).toMatchSnapshot();  
}); 