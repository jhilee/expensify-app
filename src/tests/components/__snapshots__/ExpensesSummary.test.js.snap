// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correct statments totalling 1 expenses 1`] = `
<div>
  <h3>
    Viewing 
    1
     
    expense
     totalling 
    $45.00
  </h3>
</div>
`;

exports[`should render correct statments totalling 2 expense 1`] = `
<div>
  <h3>
    Viewing 
    2
     
    expenses
     totalling 
    $560,000.34
  </h3>
</div>
`;
